packages/core/src/scuttle.js

```diff
@@ -22,6 +22,14 @@ const {
   defineProperty,
 } = Object
 
+const preserveLanguageIntrinsics = (function(cg, g) {
+  return Object.getOwnPropertyNames(cg)
+    // skip all intrinsics that a Compartment already grants
+    .filter(a => cg[a] === g[a])
+    // support LM,SES exported APIs and polyfills that LM counts on
+    .concat(['Compartment', 'Error', 'globalThis'])
```

Comment:
- what of `lockdown`/`harden`? I'm not clear on how these three global props were chosen.
- these should probably live in `src/constants.js` as a read-only array

Reply:
* They were chosen a while ago for breaking LavaMoat (or other vetted shims of MetaMask) when are scuttled (can be revisited in the future)
* They can't (not easily), since this is a static file that is not being imported dynamically, but being statically injected into output bundles (for browser oriented pkgs)